templates/overrides/semantic/web/TagHelpers/MenuLinkTagHelper.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Html.Abstractions;
+using Microsoft.AspNet.Mvc.TagHelpers;
+using Microsoft.AspNet.Mvc.ViewFeatures;
+using Microsoft.AspNet.Razor.TagHelpers;
+using Microsoft.Extensions.WebEncoders;
+
+namespace <%= namespace %>.TagHelpers
+{
```

Comment:
Here put Why's for this class. It a different one that people can see in aspnet/Teplates. for example:
```
/// <summary>
/// 
/// </summary>
```
https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.TagHelpers/LinkTagHelper.cs#L20-L26